### DISCUSSION TITLE
Imported 3D Objects disappear

### DISCUSSION THREAD

**Discussion by GinosergioZ** [mozilla/hubs#5274] [Art Questions] [answered]
Hi, I am completely new to HUBS. I Have read the FAQ, it say that your room is permanent and last forever, until you close it.
Well: using my Quest 2 I have entered a default room (that seems a museum with 3 leves, stairs, etc). In this room I import some 3D model taken from Sketchfab, I resize them and place them where I want. At the end I put away my QUEST without closing the room and even closing the browser.

When I take my QUEST later and look again at the HUBS page where I was before, I find something like "Your session is ended".  I enter again the room and all my 3D object are gone.

So, how can I have a room that lasts forever, with all objects I want ?

Thank you !
Ser

**Comment by matthewbcool** [comment, marked as answer]
Thanks for the question and trying out Mozilla Hubs! Any objects you bring into the room should be expected to leave with you unless you pin them. When you pin an object, it will persist in the room for you. 
https://hubs.mozilla.com/docs/hubs-features.html#object-menu


If you have many edits you would like to make, you can try to 'remix in spoke' from the scene link: https://hubs.mozilla.com/scenes/KV6MwkU/gathering-hall
Editing the scene itself this way will let you set up the scene however you would like.

**Reply by GinosergioZ** [reply]
Thank you Matthewbcool !!  That is a very useful Manual, I did not find it by myself. Let me know if there are other useful pages, to learn this beautiful platform !
